config/locales/exceptions.en.yml

```diff
@@ -445,6 +445,12 @@ en:
         code: 'PAGERDUTY_429'
         detail: "Exceeded PagerDuty's API rate limit"
         status: 429
+      PPMS_500:
+        <<: *external_defaults
+        title: Unexpected service error
+        code: 'PPMS_500'
+        detail: 'An unexpected error has occurred'
+        status: 500
```

Comment:
is a 500 exception from downstream necessarily a 500 exception? I'm wondering if this ought to be 502 or 503?

Reply:
Good nit, yes this should probably send a `502 - Bad Gateway` as it matches: 

> The server, while acting as a gateway or proxy, received an invalid response from an inbound server it accessed while attempting to fulfill the request.